src/v/raft/append_entries_buffer.cc

```diff
@@ -61,16 +61,12 @@ ss::future<> append_entries_buffer::stop() {
 }
 
 void append_entries_buffer::start() {
-    (void)ss::with_gate(_gate, [this] {
+    ssx::spawn_with_gate(_gate, [this] {
         return ss::do_until(
           [this] { return _gate.is_closed(); },
           [this] {
               return _enqueued.wait([this] { return !_requests.empty(); })
-                .then([this] { return flush(); })
-                .handle_exception_type(
-                  [](const ss::broken_condition_variable&) {
-                      // ignore exception, we are about to stop
-                  });
+                .then([this] { return flush(); });
```

Comment:
ahh looks much better. tougher analysis here given that broken condition could in principle bubble up from flush rather than I think what is implied at this level which it is coming from _enqueued.broken and now the loop isn't continued its aborted immediately.